ai-server/app/music/run_musicgen.py

```
@@ -1,30 +1,40 @@
-from audiocraft.data.audio import audio_write
+import os
+import re
 import tempfile
-from pathlib import Path
+from audiocraft.models import MusicGen
+
+import torchaudio
 
-# 디바이스 설정
-device = "cuda" if torch.cuda.is_available() else "cpu"
+from app.core.exceptions import AIServerError, GenerationError
```

Ruff (0.11.9)

`app.core.exceptions.AIServerError` imported but unused

Remove unused import: `app.core.exceptions.AIServerError`

(F401)